jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
It is and has been removed.
But I think we should add the [editorconfig](http://editorconfig.org/)